platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -264,8 +269,14 @@ typedef struct odp_ipsec_sa_lookup_s {
 
 /** IPSEC AAD */
 typedef struct ODP_PACKED {
-	odp_u32be_t spi;     /**< Security Parameter Index */
-	odp_u32be_t seq_no;  /**< Sequence Number */
+	/**< Security Parameter Index */
+	odp_u32be_t spi;
+
+	/**< Sequence Number */
+	union {
+		odp_u32be_t seq_no;
+		odp_u64be_t seq_no64;
```

Comment:
There is no need to have both seq_no and seq64 fiels in the state. There is only one sequence number and in the 32 bit case the high order bits would just be zero. So please, have just single uint64_t seq_no here (it is a bug that the existing seq_no is odp_u32be_t instead of uint32_t, but it does not affect compiled code).

Reply:
Will check the feasibility.

Reply:
yes, changed the "odp_u32be_t seq_no" to "uint64_t seq_no" and removed "odp_u64be_t seq_no64".

Reply:
I wrote: "There is no need to have both seq_no and seq64 fiels in the state."

So I was talking about the sequence numbers in the ipsec_state_t, not in ipsec_aad_t, but unfortunately I placed the comment in the wrong place in the github interface. Sorry about that. Now in the latest revision the state struct still has two sequence number fields and the change in ipsec_aad_t introduces a byteorder bug and should not have been done.

Reply:
> I wrote: "There is no need to have both seq_no and seq64 fiels in the state."
So I was talking about the sequence numbers in the ipsec_state_t, not in ipsec_aad_t, but unfortunately I placed the comment in the wrong place in the github interface. Sorry about that. 

Thought it "in the state" was a typo, so made changes in "ipsec_aad_t".  
Next
1) will revert the ipsec_aad_t" changes.
2) In "ipsec_state_t" will just use the "uint64_t seq_no" and will delete the seq_no64.

Reply:
> will revert the ipsec_aad_t" changes.
> In "ipsec_state_t" will just use the "uint64_t seq_no" and will delete the seq_no64.
> 

These above mentioned changes are made.